<ui version="4.0" >
 <class>AddContact</class>
 <widget class="QWidget" name="AddContact" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>240</width>
    <height>173</height>
   </rect>
  </property>
  <property name="sizePolicy" >
   <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle" >
   <string>Add User</string>
  </property>
  <property name="windowIcon" >
   <iconset>../../../../../../../:/icons/crystal_project/search.png</iconset>
  </property>
  <widget class="QLabel" name="label" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>10</y>
     <width>56</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text" >
    <string>Jabber ID:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>40</y>
     <width>56</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text" >
    <string>Name:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2" >
   <property name="geometry" >
    <rect>
     <x>10</x>
     <y>70</y>
     <width>56</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Group:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="authorizationCheck" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>90</y>
     <width>256</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text" >
    <string>Send authorization request</string>
   </property>
   <property name="checked" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="nickName" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>40</y>
     <width>171</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="jidName" >
   <property name="geometry" >
    <rect>
     <x>61</x>
     <y>11</y>
     <width>131</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="groupName" >
   <property name="geometry" >
    <rect>
     <x>60</x>
     <y>70</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="editable" >
    <bool>true</bool>
   </property>
   <item>
    <property name="text" >
     <string>&lt;no group></string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="buttonAdd" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>110</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Add</string>
   </property>
   <property name="icon" >
    <iconset>../../../../../../../:/icons/crystal_project/apply.png</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonFind" >
   <property name="geometry" >
    <rect>
     <x>70</x>
     <y>110</y>
     <width>81</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Find user</string>
   </property>
   <property name="icon" >
    <iconset>../../../../../../../:/icons/crystal_project/search.png</iconset>
   </property>
  </widget>
  <widget class="QToolButton" name="buttonInfo" >
   <property name="geometry" >
    <rect>
     <x>210</x>
     <y>10</y>
     <width>26</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text" >
    <string/>
   </property>
   <property name="icon" >
    <iconset>../../../../../../../:/icons/crystal_project/contactinfo.png</iconset>
   </property>
   <property name="autoRaise" >
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="buttonCancel" >
   <property name="geometry" >
    <rect>
     <x>160</x>
     <y>140</y>
     <width>75</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text" >
    <string>Cancel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>groupName</sender>
   <signal>textChanged(QString)</signal>
   <receiver>AddContact</receiver>
   <slot>on_groupText_changed(QString)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>128</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel" >
     <x>56</x>
     <y>122</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
